### PR TITLE
Application Integration dashboards

### PR DESCRIPTION
**Pull request by ShubhamThakkar**

### CODE REVIEW

File: dashboards/google-application-integration/integration-executions.01.png
Comment (yqlu) [inline dashboards/google-application-integration/integration-executions.01.png:1]: General comment that applies to all the screenshots here -- is it possible to mock data such that all the charts in the screenshots have data populated? These are used as preview images, so if they're populated, it's easier to showcase the dashboard (e.g. which charts are line charts, stacked area charts, bar charts, what are the y axes, ...)

File: dashboards/google-application-integration/integration-executions.json [hunk 1]
```diff
@@ -1,501 +1,417 @@
...
-                      }
-                    }
+        "width": 4,
+        "xPos": 8,
+        "yPos": 0
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "Error (atleast one) containing executions per min",
```
Comment (yqlu) [inline dashboards/google-application-integration/integration-executions.json:288]: atleast -> at least

File: dashboards/google-application-integration/integration-executions.01.png
Comment (yqlu) [reply dashboards/google-application-integration/integration-executions.01.png:1]: This also serves as a verification that there are no typos or errors in the filter or query populating the chart

File: dashboards/google-application-integration/integration-executions.json (hunk 1 above)
Comment (yqlu) [reply dashboards/google-application-integration/integration-executions.json:288]: Reading this again, I think the original phrasing was clearer? Executions with at least one error per min

File: dashboards/google-application-integration/integration-executions.01.png
Comment (ShubhamThakkar) [reply dashboards/google-application-integration/integration-executions.01.png:1]: Got it, thanks for the suggestion, have made the suggested changes.

One more thing - Is it possible to remove the process of attaching screenshots, it seems too lengthy. For every change, we have to take the screenshots again. Or somehow there should be a process to auto-generate these and show it to customer.

File: dashboards/google-application-integration/integration-executions.json (hunk 1 above)
Comment (ShubhamThakkar) [reply dashboards/google-application-integration/integration-executions.json]: Changed it.

File: dashboards/google-application-integration/data-processed.json [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    "dashboardFilters": [
+      {
+        "filterType": "RESOURCE_LABEL",
+        "labelKey": "location",
+        "templateVariable": ""
+      },
+      {
+        "filterType": "RESOURCE_LABEL",
+        "labelKey": "integration",
+        "stringValue": "shubham-monitoring-integration",
```
Comment (yqlu) [inline dashboards/google-application-integration/data-processed.json:12]: Omit this line

**Review by yqlu — APPROVED** [review]
Just one small change, then LGTM

File: dashboards/google-application-integration/integration-executions.01.png
Comment (yqlu) [reply dashboards/google-application-integration/integration-executions.01.png:1]: Unfortunately, screenshots are an integral part of how sample and integration dashboards are presented in the UI. We are trying to keep the churn to a minimum by only mandating screenshots be provided for the first submission of each dashboard and when there are major layout changes, and not when you're just making minor edits to an existing dashboard. But it's hard to auto-generate them on the backend when (as we discussed) you need to be running a workload to generate values for some of these metrics.

File: dashboards/google-application-integration/data-processed.json (hunk 2 above)
Comment (ShubhamThakkar) [reply dashboards/google-application-integration/data-processed.json]: Removed it, thanks for catching it.

Thanks for reviewing the CL.

**Pull request merged**